contributing/changelog-process.md

```diff
@@ -151,3 +151,8 @@ A PR check is included to validate the changelog entry file.
 If a PR doesn't need a changelog entry its check can be skipped:
 - Adding the label `skip-changelog-check` to the PR.
 - Check in PRs with title `chore`, `test`, `doc` or `ci` is automatically skipped. However a changelog can still be added if needed.
+
+## Unreleased section of CHANGELOG.md automatic update
+
+After a PR is merged to with a new entry in `.changelog` directory, [Generate CHANGELOG workflow](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/workflows/generate-changelog.yml) will be triggered and it will update the `CHANGELOG.md` file with the new entry.
```

Comment:
```suggestion
After a PR is merged to master with a new entry in `.changelog` directory, [Generate CHANGELOG workflow](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/workflows/generate-changelog.yml) will be triggered and it will update the `CHANGELOG.md` file with the new entry.
```

Reply:
This made me realise that with current implementation, the action would be triggered if a PR is merged into any branch (not only master). Thanks! [Fixed here](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2128/commits/25e59b9483d71887ca1fcde827f4828671d10af0)

Reply:
what about doing something similar to examples.yaml and run on push instead of pull_request:

push:
    branches:
      - master